### PR TITLE
Replace usernames with a team name in CODEOWNERS

### PR DESCRIPTION
**Pull request by kasiakepka**
**Description**

Changes proposed in this pull request:

- Replace usernames with a team name
**Related issue(s)**

### CODE REVIEW

**Comment by a-thaler** [comment]
I recommend to use group name similar to the domain/are the people are responsible for

**Comment by kasiakepka** [comment]
Thanks Andreas, we will consider that but I am afraid that we have to wait for Magda (till Monday) before we make final decision.

**Review by pbochynski — APPROVED** [review]

**Pull request merged**
